### ISSUE TITLE
rustup still fail to install stable rust on ppc64le (even 1.26.1)

### ISSUE THREAD

**Issue by lu-zero** [rust-lang/rust#51199]
```
# rustup stable install
info: syncing channel updates for 'stable-powerpc64le-unknown-linux-gnu'
info: latest update on 2018-05-29, rust version 1.26.1 (827013a31 2018-05-25)
error: component 'rust-docs' for 'powerpc64le-unknown-linux-gnu' is unavailable for download
```

`ppc64be` seems to install fine.

**Comment by pietroalbini** [comment]
`rustup stable install` is not a valid rustup command (at least on the latest version). What's the exact command you're using?

**Comment by lu-zero** [comment]
Wiping completely rustup and installing from scratch worked.

**Comment by pietroalbini** [comment]
Yeah, that's a known issue with rustup (it can install the versions fixed by 1.26.1 but it can't update to them). [A rustup patch was merged](https://github.com/rust-lang-nursery/rustup.rs/pull/1419), and it will be available in the next release.
